src/librustdoc/html/static/js/search.js

```diff
@@ -5328,12 +5319,12 @@ async function search(forced) {
     // recent search query is added to the browser history.
     updateSearchHistory(buildUrl(query.userQuery, filterCrates));
 
+    console.log(filterCrates);
+
```

Comment:
```suggestion
```

Reply:
whoops!

yea this was from testing to figure out which type signature was correct.